src/atoms/Input/Input.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import InputAtom from './Input';
+
+describe('Input Atom works properly', () => {
+  it('has the correct width', () => {
+    render(<InputAtom placeholder="test" />);
+    expect(screen.getByPlaceholderText('test')).toHaveStyle('width: 25ch');
+  });
+});
```

Comment:
I would like to see some test in regards of the `onKeyUp` function. Also, remember to add the `PropTypes` to your components, this way you make them more resilient to changes. It looks like you have some required properties for your component, but not taking them into account for validations